timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/AcceptorCacheDigest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.paxos;
+
+import java.util.Map;
+import java.util.UUID;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableAcceptorCacheDigest.class)
+@JsonSerialize(as = ImmutableAcceptorCacheDigest.class)
+public interface AcceptorCacheDigest {
+    UUID newCacheKey();
+    Map<Client, Long> updates();
```

Comment:
nit: might be good to define what happens if an acceptor receives new proposals at multiple sequence numbers for a given client? (Specifically, either in doc or parameter name, that it returns the latest in that case?)